docs/docs/build/presence.md

```diff
@@ -110,6 +110,42 @@ Notifications API is partially implemented. All messages are always broadcast ev
 
 Notifications are fundamentally unreliable at this time as there are no built-in acknowledgements nor retained state. To prevent most common loss of notifications, always check for connection before sending.
 
-### Throttling
+### Throttling/grouping
 
-Throttling is not yet implemented. `BroadcastControls` exists in the API to provide control over throttling of value updates, but throttling is not yet implemented. It is recommended that `BroadcastControls.allowableUpdateLatencyMs` use is considered and specified to light up once support is added.
+Presence updates are grouped together and throttled to prevent flooding the network with messages when presence values are rapidly updated. This means the presence infrastructure will not immediately broadcast updates but will broadcast them after a configurable delay.
+
+The `allowableUpdateLatencyMs` property configures how long a local update may be delayed under normal circumstances, enabling grouping with other updates. The default `allowableUpdateLatencyMs` is **60 milliseconds** but may be (1) specified during configuration of a [States Workspace](#states-workspace) or [Value Manager](#value-managers) and/or (2) updated later using the `controls` member of Workspace or Value Manager. [States Workspace](#states-workspace) configuration applies when a Value Manager does not have its own setting.
+
+Notifications are never queued; they effectively always have an `allowableUpdateLatencyMs` of 0. However, they may be grouped with other updates that were already queued.
+
+Note that due to throttling, clients receiving updates may not see updates for all values set by another. For example,
+with `Latest*ValueManagers`, the only value sent is the value at the time the outgoing grouped message is sent. Previous
+values set by the client will not be broadcast or seen by other clients.
+
+#### Example
+
+You can configure the grouping and throttling behavior using the `allowableUpdateLatencyMs` property as in the following example:
+
+```ts
```

Comment:
Not for this PR, but this reminded me: I'm trying to establish a pattern for maintaining "living" code examples for use in situations like this. I'd be interested in your thoughts. Example in `api-markdown-documenter`:
https://github.com/microsoft/FluidFramework/blob/main/tools/api-markdown-documenter/README.md?plain=1#L71

Reply:
I like living examples. To be most effective they would have to be tied to the released code. But most docs are slightly divorced from what is released. That means updating after or with release and we'll mess it up. I support figuring it all out for sure.

Reply:
> But most docs are slightly divorced from what is released.

True in a lot of cases, but for package-local documentation like this, the two should generally be in-sync.

Reply:
Main docs are from main branch and API docs are from latest released branch. That 2-to-3-week possible disparity is what you mean by "generally in-sync"?